tests/experimental_combination/attenuator_test.py

```diff
@@ -8,168 +11,214 @@
-        {
-            "avl_location": {
-                "position": {
-                    "latitude": 39.739928,
-                    "longitude": -104.593591,
-                    "bearing": 45,
-                    "speed": 5,
-                    "odometer": None
-                },
-            },
-            "rtdh_message_id": "110e5ecc-cd7f-48dc-8e59-ac6ef49fc1d0",
-            "rtdh_timestamp": "2022-07-28T21:05:53Z"
-        }]
-    wzdx_msgs = [
-        {
-            'features': [
-                {
-                    'id': 42,
-                    'geometry': {
-                        'coordinates': [
-                            [-104.599491, 39.740070],
-                            [-104.584285, 39.739899]
-                        ]}
-                }
-            ]
-        }
-    ]
-
-    features = attenuator.identify_overlapping_features(geotab_msgs, wzdx_msgs)
-    assert len(features) == 1
-
-
-@patch.object(attenuator, 'get_geometry_for_distance_ahead')
-def test_combine_with_wzdx_ordered(atten_patch):
-    attenuator.get_geometry_for_distance_ahead = MagicMock(
-        return_value=([], 0, 1))
-    wzdx_feature = {'properties': {}, 'geometry': {}}
-    route_details = {}
-    distance_ahead = 0
-    bearing = 0
-    event_start_marker = 5
-    event_end_marker = 10
-    actual = attenuator.combine_with_wzdx(
-        wzdx_feature, route_details, distance_ahead, bearing, event_start_marker, event_end_marker)
-
-    expected = {'properties': {
-        'beginning_milepost': 0,
-        'ending_milepost': 1,
-    },
-        'geometry': {
-        'coordinates': [],
-    }}
-
-    attenuator.get_geometry_for_distance_ahead.assert_called_with(
-        distance_ahead, route_details, bearing, event_start_marker, event_end_marker)
-
-    assert actual == expected
-
-
-@patch.object(attenuator, 'get_geometry_for_distance_ahead')
-def test_combine_with_wzdx_reversed(atten_patch):
-    attenuator.get_geometry_for_distance_ahead = MagicMock(
-        return_value=([], 0, 1))
-    wzdx_feature = {'properties': {}, 'geometry': {}}
-    route_details = {}
-    distance_ahead = 0
-    bearing = 0
-    event_start_marker = 10
-    event_end_marker = 5
-    actual = attenuator.combine_with_wzdx(
-        wzdx_feature, route_details, distance_ahead, bearing, event_start_marker, event_end_marker)
-
-    expected = {'properties': {
-        'beginning_milepost': 0,
-        'ending_milepost': 1,
-    }, 'geometry': {
-        'coordinates': [],
-    }}
-
-    attenuator.get_geometry_for_distance_ahead.assert_called_with(
-        distance_ahead, route_details, bearing, event_end_marker, event_start_marker)
-
-    assert actual == expected
-
-
-@patch.object(cdot_geospatial_api, 'get_route_geometry_ahead', side_effect=[{'coordinates': 'a', 'start_measure': 'b', 'end_measure': 'c'}])
+    assert len(combined_events[0]["features"][0]["geometry"]["coordinates"]) > 2
+
+
+# def test_identify_overlapping_features_valid():
```

Comment:
Are these invalid tests or just failing? If just failing, do we need to address to fix? If invalid, can we remove?

Reply:
These were a little out of date, my initial reaction was to comment them and come back later. I decided to re-write them now, so they should be good to go